Formula/d/djlint.rb

```diff
@@ -0,0 +1,92 @@
+class Djlint < Formula
+  include Language::Python::Virtualenv
+
+  desc "Lint & Format HTML Templates"
+  homepage "https://djlint.com"
+  url "https://files.pythonhosted.org/packages/a4/ee/ac8ee551ec04d5a214e62d008f40bf309574f103416482e03fb61fbad61c/djlint-1.35.2.tar.gz"
+  sha256 "318de9d4b9b0061a111f8f5164ecbacd8215f449dd4bd5a76d2a691c815ee103"
+  license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

https://github.com/djlint/djLint/blob/d9aaac591efab40d3d2c878e0154fc0f91658ab3/pyproject.toml#L18

Reply:
Thank you! I'll commit it once the checks are done.